CHANGELOG.md

```diff
@@ -4,6 +4,41 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [2.0.8] - 2023-07-31
+
+### Changed
+- Removed phusion-openldap-builder, phusion-ruby-builder and phusion-ruby-fips.
+  Removed references to general Phusion. Removed with references to them:
+  openssl-builder, postgres-client-builder and ubuntu-ruby-builder.
+- Added nginx to the base image of ubuntu-ruby-postgres-fips.
+- Added Runit to Ubuntu image.
+- Generalized test cases in ubi and ubuntu images.
```

Comment:
The changelog is generally reserved for changes that have an impact on the end-user. Changes to tests or for developer productivity should be left out to keep it clear what's changed for the consumers of the release for each version.